scribus/ui/smcstylewidget.cpp

```diff
@@ -191,30 +191,30 @@ void SMCStyleWidget::fillColorCombo(ColorList &colors)
 	backColor_->view()->setMinimumWidth(backColor_->view()->maximumViewportSize().width()+24);
 }
 
-void SMCStyleWidget::handleUpdateRequest(int updateFlags)
-{
-	if (!m_Doc)
-		return;
-	if (updateFlags & reqColorsUpdate)
-		fillColorCombo(m_Doc->PageColors);
+void SMCStyleWidget::handleUpdateRequest(int updateFlags)
+{
+	if (!m_Doc)
+		return;
+	if (updateFlags & reqColorsUpdate)
+		fillColorCombo(m_Doc->PageColors);
 }
 
-void SMCStyleWidget::setDoc(ScribusDoc *doc)
-{
-	if (m_Doc)
-		disconnect(m_Doc->scMW(), SIGNAL(UpdateRequest(int)), this , SLOT(handleUpdateRequest(int)));
-
-	m_Doc = doc;
-	if (!m_Doc)
-		return;
-
-	QStringList languageList;
+void SMCStyleWidget::setDoc(ScribusDoc *doc)
+{
+	if (m_Doc)
+		disconnect(m_Doc->scMW(), SIGNAL(UpdateRequest(int)), this , SLOT(handleUpdateRequest(int)));
+
+	m_Doc = doc;
+	if (!m_Doc)
+		return;
+
+	QStringList languageList;
 	LanguageManager::instance()->fillInstalledStringList(&languageList, false);
-	fillLangComboFromList(languageList);
-	fillColorCombo(m_Doc->PageColors);
-	fontFace_->RebuildList(m_Doc);
-
-	connect(m_Doc->scMW(), SIGNAL(UpdateRequest(int)), this , SLOT(handleUpdateRequest(int)));
+	fillLangComboFromList(languageList);
+	fillColorCombo(m_Doc->PageColors);
+	fontFace_->RebuildList(m_Doc);
+
+	connect(m_Doc->scMW(), SIGNAL(UpdateRequest(int)), this , SLOT(handleUpdateRequest(int)));
```

Comment:
I really tired many times to remove it but I couldn't. I don't see where I can edit and I don't remember I edited this part.